riscof/framework/main.py

```diff
@@ -126,10 +126,15 @@ def run_coverage(base, dut_isa_spec, dut_platform_spec, work_dir, cgf_file=None)
                             'test_size': [str(entry) for entry in find_elf_size(elf)],
                             'test_groups': str(set(test_list[entry[0]]['coverage_labels']))
                             })
+    flen = 0
+    if 'F' in ispec['ISA']:
+        flen = 32
+    elif 'D' in ispec['ISA']:
+        flen = 64
```

Comment:
Support for this flen is already do exist, as i understood from [here](https://github.com/riscv-software-src/riscof/pull/44#issuecomment-1101971805). If so can't use the same method across, rather two different ways.

Reply:
It is the same logic. The FLEN is derived from the isa of the dut. ([ref](https://github.com/riscv-software-src/riscof/blob/66bb6c468afaa3775ab5a757459a560ceaa5ee8c/riscof/framework/test.py#L368))